README.md

```diff
@@ -3,7 +3,6 @@ page_type: sample
 languages:
 - csharp
 products:
-- msal-net
```

Comment:
Let's use msalnet instead, I see its on the [taxonomies allowlist](https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#product) under dotnet

Reply:
changed to msalnet